kani-compiler/src/codegen_cprover_gotoc/codegen/assert.rs

```diff
@@ -45,6 +45,10 @@ pub enum PropertyClass {
     ///
     /// SPECIAL BEHAVIOR: "Errors" for this type of assertion just mean "reachable" not failure.
     Cover,
+    /// See [GotocCtx::codegen_coverage] below. Always an `assert(false)` that's not an error.
+    ///
+    /// SPECIAL BEHAVIOR: "Errors" for this type of assertion just mean "reachable" not failure.
+    Coverage,
```

Comment:
Can you please document the difference between `Cover` and `Coverage`? How is someone supposed to know when to use one over the other?

Reply:
I've added the main differences here and made clear the use case.